### PR TITLE
Correct inconsistencies in notebook for proj onboarding.

### PR DESCRIPTION
**Pull request by HumairAK**
As per: https://github.com/operate-first/SRE/issues/417#issuecomment-981802687

- I removed a lot of information that isn't necessary to the user and just adds messy documentation. 
- User group is now added to the common folder. 
- Also added ability to add a standard or custom quota
- A list of clusters is also made available, dynamically generated. 
- Kustomization files are sorted after being updated.

### CODE REVIEW

File: pages/onboarding_project.ipynb [hunk 1]
```diff
@@ -80,146 +130,413 @@
...
+    "%cd {WORKDIR}/cluster-scope/base/core/namespaces/{NAMESPACE_NAME}\n",
+    "if(not CUSTOM_QUOTA): \n",
+    "    !kustomize edit add component ../../../../components/resourcequotas/{QUOTA}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "If you require a custom quota, please enter the values for the resources you require:"
```
Comment (4n4nd) [inline pages/onboarding_project.ipynb:304]: Should we add a line that tells user to skip to step 6 if they don't need a custom quota?

**Review by schwesig — APPROVED** [review]

File: pages/onboarding_project.ipynb (hunk 1 above)
Comment (HumairAK) [reply pages/onboarding_project.ipynb:304]: nah that's why I put an `if` clause, so it just won't count.

**Pull request merged**